test/linear_samplers/proc_block_col_FJLT.jl

```diff
@@ -0,0 +1,63 @@
+# This file is part of RLinearAlgebra.jl
+
+module ProceduralTestLSBCFJLT
```

Comment:
LOL. I am so sorry I ever did this. SBCFJLT is the by abbreviation I have ever seen.

Reply:
Yeah the abbreviations do start to get very cumbersome